test/drivers/open_api_3_test.rb

```diff
@@ -0,0 +1,82 @@
+require_relative "../test_helper"
+
+describe Committee::Drivers::OpenAPI3 do
+  before do
+    @driver = Committee::Drivers::OpenAPI3.new
+  end
+
+  it "has a name" do
+    assert_equal :open_api_3, @driver.name
+  end
+
+  it "has a schema class" do
+    assert_equal Committee::Drivers::OpenAPI3::Schema, @driver.schema_class
+  end
+
+  it "override methods" do
+    parser = OpenAPIParser.parse(open_api_3_data)
+    schema = @driver.parse(parser)
+
+    assert_kind_of Committee::Drivers::OpenAPI3::Schema, schema
+    assert_kind_of OpenAPIParser::Schemas::OpenAPI, schema.open_api
+    assert_equal @driver, schema.driver
+  end
+
+  it "defaults to coercing form parameters" do
+    assert_equal true, @driver.default_coerce_form_params
+  end
+
+  it "defaults to path parameters" do
+    assert_equal true, @driver.default_path_params
+  end
+
+  it "defaults to query parameters" do
+    assert_equal true, @driver.default_query_params
+  end
+
+  describe "Schema" do
+    describe "#operation_object" do
+      describe "path template" do
+        it "get normal path" do
```

Comment:
Can we change this and all instances below of `it "get ...` to `it "gets a ...`?